pages/common/poetry.md

```diff
@@ -26,3 +26,11 @@
 - Execute a command inside the project's virtual environment:
 
 `poetry run {{command}}`
+
+- Bump the minor version of the project in `pyproject.toml`:
+
+`poetry version minor`
+
+- List all commands:
```

Comment:
This makes it a little clearer.
```suggestion
- List all poetry subcommands:
```

Reply:
Ok thanks, to accept the suggestion do I just click commit suggestion displayed here with the default message it gives or should I name the commit something in particular?